tests/drivers/clock_control/stm32_clock_configuration/stm32u5/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 cmake_minimum_required(VERSION 3.20.0)
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(stm32_clock_configuration)
+project(stm32_clock_configuration_h5)
```

Comment:
Is the `project(stm32_clock_configuration_h5) `or project(stm32_clock_configuration_**u**5)

Reply:
Right, this is now fixed. Txs

Reply:
Still need to be changed to stm32_clock_configuration_u5